index.html

```diff
@@ -30,7 +30,7 @@ <h1>Get ready to Spot the Surveillance...</h1>
         <img id="congrats-card" src="assets/images/congrats-card-replay.png">
         <img id="drone-card"  src="assets/images/drone-card.png">
         <img id="intro-card" src="assets/images/intro-card.png">
-        <img id="eff-logo" src="assets/images/eff-logo-black.svg">
```

Comment:
@lschatzkin: can we remove the `eff-logo-black.svg` from this repo?

Reply:
> @lschatzkin: can we remove the `eff-logo-black.svg` from this repo?

yes!